deploy/fb-submitter-chart/templates/cronjob_db_sweeper.yaml

```diff
@@ -49,4 +49,14 @@ spec:
                   secretKeyRef:
                     name: fb-submitter-app-secrets-{{ .Values.environmentName }}
                     key: sentry_dsn
+              - name: encryption_key
+                valuefrom:
+                  secretkeyref:
+                    name: fb-submitter-app-secrets-{{ .values.environmentname }}
```

Comment:
Is that supposed to be all lowercase? At line 50 it's mixed

Reply:
Yikes good catch